cli.js

```diff
@@ -28,6 +29,12 @@ const cli = meow(`
 
 updateNotifier({pkg: cli.pkg}).notify();
 
+const pkg = readPkgUp.sync().pkg;
```

Comment:
Actually, `cli.pkg` already has the package data. It uses [read-pkg-up](https://github.com/sindresorhus/meow/blob/master/index.js#L24) as well.